govuk-one-login/authentication-api
# BAU: It's Never Localstack

## What

It's never localstack. Remove all references to localstack

## How to review

1. Deploy to dev
2. Check the deploy worked
